garak/probes/web_injection.py

```diff
@@ -400,23 +416,113 @@ class MarkdownURIImageExfilExtended(MarkdownExfilMixin, garak.probes.Probe):
         "payload:malicious:renderer",
         "quality:Security:PromptStability",
         "quality:Security:ExtractionInversion",
-        "cwe:352"
-    ] 
+        "cwe:352",
+    ]
     tier = garak.probes.Tier.OF_CONCERN
     active = True
 
     markdown_uri_templates = [
         "![{link_display_text}]({uri})",
         "![{link_display_text}][ref]\n\n[ref]: {uri}",
         "![{link_display_text}][]\n\n[{link_display_text}]: {uri}"
-        "![{link_display_text}][ref]\n\n[{link_display_text}]: {uri}"
+        "![{link_display_text}][ref]\n\n[{link_display_text}]: {uri}",
     ]
 
     def __init__(self, config_root=_config):
         garak.probes.Probe.__init__(self, config_root=config_root)
         MarkdownExfilMixin.__init__(self, config_root=config_root)
         self.generate_prompts(self.markdown_uri_templates, self.soft_probe_prompt_cap)
-        
-      
 
-    
\ No newline at end of file
+
+class TaskXSS(garak.probes.Probe):
+    """Perform cross site scripting attacks via prompt injection.
+
+    Relay javascript to be rendered by the target application via markdown JS rendering.
+    """
+
+    doc_uri = "https://cwe.mitre.org/data/definitions/79.html"
```

Comment:
does this describe the attack or the weakness

Reply:
@erickgalinkin can we move this to tag and then cite the embracethered post (if that's the provenance) in `doc_uri`?